internal/db/fetcher/versioned.go

```diff
@@ -377,32 +387,25 @@ func (vf *VersionedFetcher) merge(c cid.Cid) error {
 }
 
 func (vf *VersionedFetcher) processBlock(
```

Comment:
suggestion: To make the function name more representative of what it has become, it would make sense to rename it to `processFieldBlock`. This way there will be no confusion as to what it's for.

Reply:
Do you mind if I remove this function entirely?

Reply:
I don't.

Reply:
Thanks :)